src/DataFrame/Operations/Statistics.hs

```diff
@@ -197,4 +197,23 @@ computeVariance (VarAcc n _ m2)
     | otherwise = m2 / fromIntegral (n - 1)
 
 variance' :: VU.Vector Double -> Double
-variance' = computeVariance . VG.foldl' step (VarAcc 0 0 0)
+variance' = computeVariance . VG.foldl' varianceStep (VarAcc 0 0 0)
+
+data SkewAcc = SkewAcc !Int !Double !Double !Double deriving (Show)
+
+skewnessStep :: SkewAcc -> Double -> SkewAcc
+skewnessStep (SkewAcc !n !m !m2 !m3) !x =
+    let !k   = fromIntegral n
+        !n'  = n + 1
+        !m'  = m  + (x - m) / k
+        !m2' = m2 + ((x - m) ^ 2 * (k - 1)) / k
+        !m3' = m3 + ((x - m) ^ 3 * (k - 1) * (k - 2)) / k ^ 2 - (3 * (x - m) * m2) / k
+     in SkewAcc n' m' m2' m3'
+
+computeSkewness :: SkewAcc -> Double
+computeSkewness (SkewAcc n _ m2 m3)
+    | n < 3 = 0 -- or error "skewness of <3 samples"
```

Comment:
The statistics package returns NaN. Which is `read "NaN" :: Double)`. I don't know if I like that. Zero might be better. I'll leave this up to your discretion.